src/main/java/io/reactiverse/awssdk/converters/VertxAsyncResponseTransformer.java

```diff
@@ -0,0 +1,42 @@
+package io.reactiverse.awssdk.converters;
+
+import io.reactiverse.awssdk.reactivestreams.WriteStreamSubscriber;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.streams.WriteStream;
+import software.amazon.awssdk.core.async.AsyncResponseTransformer;
+import software.amazon.awssdk.core.async.SdkPublisher;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.CompletableFuture;
+
+public class VertxAsyncResponseTransformer<ResponseT> implements AsyncResponseTransformer<ResponseT, WriteStream<Buffer>> {
+
+    private volatile CompletableFuture<WriteStream<Buffer>> cf;
+    private volatile ResponseT response;
+    private volatile WriteStream<Buffer> writeStream;
+
```

Comment:
as described in #35 a handler that will be called when the response is available, but the stream has not been written to would be great:

`private Handler<ResponseT> responseHandler;`